user-profiles.md

```diff
@@ -0,0 +1,33 @@
+# User Profiles
+
+When considering requirements and implementation solutions it's useful look at who uses Helm to have a better understanding of how something should work. The Helm user profiles describe the different types of users and contributors to Helm along with the order of priority they have relative to each other. The ordering is because nothing can share the exact same priority and we want to convey that.
+
+## Profiles
+
+Profiles describe a type of user. A real person may perform more than one type of role.
+
+### 1. Application Operator
+
+Application operators take an application and operate it within a Kubernetes cluster. For example, the operation of Wordpress and MySQL.
+
+### 2. Application Distributor
+
+Distributors are someone who packages an application for someone else to operate. Examples of this include Wordpress, with a MySQL dependency, and Datadog.
+
+### 3. Application Developer
+
+An application developer writes an application that can be operated in one or more environment. They are typically not concerned with the where the application is running. That is whether it's a container or virtual machine. Examples of this include the developers of Wordpress and MySQL.
+
+### 4. Supporting Tool Developer
+
+Supporting tool developers build tools adjacent to Helm such as a linter, Helm plugin, or even kubectl. These are developers building complementary things that can be used along with Helm.
+
+### 5. Helm Developer
+
+Helm developers are those who develop Helm itself. That includes core maintainers along with anyone else who fixes a bug or updates docs.
+
+Generally speaking, the developers of Helm consider the end users above themselves when looking at requirements and implementation strategies.
+
+## Notes
```

Comment:
Perhaps this would be better as a preface to the document rather than a footnote?

Reply:
Good idea. I moved it.